### DISCUSSION TITLE
SOC page not loading in 2.3.21

### DISCUSSION THREAD

**Discussion by rayrivera** [Security-Onion-Solutions/securityonion#2498] [Unsupported Versions] [answered]
Hello, I am not able to connect to the SOC page with either ip or hostname. I am running the newest version of SecurityOnion 2.3.21 and have allowed the ip addresses in so-allow firewall. Using Chrome the webpage comes up and states it's not secure then when I select connect anyways nothing happens. Thanks for any help.

**Comment by dougburks** [comment]
What did you select at this Setup prompt?
![image](https://user-images.githubusercontent.com/1659467/103364892-d424f200-4a8c-11eb-8470-1d1042edadad.png)

**Reply by rayrivera** [reply]
Doug, I selected hostname and when I access the console it says 
"Access the Security Onion web interface at https://so-eval", I enter https://so-eval into Chrome and it states the site can't be reached. I have also tried it with the ip address just in case but with the same result.

**Comment by dougburks** [comment, marked as answer]
If you want to access via hostname `so-eval`, then that hostname will have to be resolvable via DNS or hosts file entry on the Chrome machine.

**Reply by rayrivera** [reply]
Doug, thank you. I was able to edit the hosts file in C:\Windows\System32\drivers\etc\hosts.  I added the address and hostname then viola, it works.
Thanks again.
